docs/user-guide/multiple_sources.md

```diff
@@ -0,0 +1,62 @@
+# Multiple Sources for an Application
```

Comment:
Should we document what happens if more than one source contains a resource with the same name/namespace? Specifically I'd want to know 1) which resource "wins" and 2) are the resources de-duplicated _before_ applying, or will there be some moment between when the _first_ version of a resource is applied and when the _second_ (or third etc.) version of a resource is applied?